handbook/brand.md

```diff
@@ -438,6 +438,142 @@ In Figma:
 
 Every week, we run `npm audit --only=prod` to check for vulnerabilities on the production dependencies of fleetdm.com. Once we have a solution to configure GitHub's Dependabot to ignore devDependencies, this manual process can be replaced with Dependabot.
 
+## Article formatting guidelines
+
+To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in Markdown (todo).
+
+#### On this page
+- [Layout](#layout)
+- [Images and screenshots](#images-and-screenshots)
+- [Meta tags](#meta-tags)
+- [Customizable CTA](#customizable-cta)
+- [Other pages of interest](#other-pages-of-interest)
+
+### Layout
+The following layout guide aims to help you create consistently formatted articles. For an existing article example, check out the [Markdown](https://raw.githubusercontent.com/fleetdm/fleet/main/articles/tales-from-fleet-security-speeding-up-macos-updates-with-nudge.md) and the [finished result](https://fleetdm.com/securing/tales-from-fleet-security-speeding-up-macos-updates-with-nudge).
+
+#### Hero image
+Consider adding a hero image for a more significant impact. Get in touch with Digital Experience via #content on Slack to make a request. 
+
+#### Table of contents
+For long articles or guides, consider adding a table of contents.
+
+#### Introduction
+It’s good practice to start your article with a clear summary of what you will be discussing.
+
+#### Main content
+The main body of your article.
+
+#### Conclusion
+It’s a good idea to finish your article with a clear closing statement.
+
+#### Add a customizable CTA
+Add a CTA at the end of your article. See [Customizable CTA](#customizable-cta) below for instructions on creating a CTA tailored to your article topic.
+
+### Images and screenshots
+Images are a great way to help engage your readers. But consider the following before including images or screenshots in your article:
+
+- Does the image add value?
+- Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.
+
+### Meta tags
+These tags help … summary of what these are for (todo)
```

Comment:
```suggestion
These tags help pass information to the website about the article to display as well as store it. 
```

Reply:
@Desmi-Dizney. You hit **Resolve conversation** again instead of **Commit suggestion** 🧐

Reply:
🤦🏻‍♂️